language/stdlib/modules/LibraConfig.move

```diff
@@ -180,6 +257,23 @@ module LibraConfig {
         );
     }
 
+    spec fun emit_reconfiguration_event {
+        pragma addition_overflow_unchecked = true;
+        // pragma opaque = true;
+        // TODO (dd): Should this warn if modifies are incomplete?
+        //    Is it dangerous to make this opaque with incomplete modifies?
```

Comment:
This is a good question. 

Currently we treat modifies like aborts_if -- if we do not provide one, anything is allowed. This allows us to incrementally add modifies (and aborts_if).

Perhaps when a function is declared *opaque*, we should choose as a default that both modifies and aborts_if are strict. 

I don't know whether this is possible for modifies, it is definitely for aborts_if. @shazqadeer @emmazzz